compiler/optimizer/OMROptimizer.hpp

```diff
@@ -204,6 +204,10 @@ class Optimizer
    void setCantBuildGlobalsUseDefInfo(bool v) { _cantBuildGlobalsUseDefInfo = v; }
    void setCantBuildLocalsUseDefInfo(bool v)  { _cantBuildLocalsUseDefInfo = v; }
 
+   TR_UseDefInfo* createUseDefInfo(TR::Compilation* comp,
```

Comment:
It is a general rule that new APIs need to have doxygen associated with them. I think it would be good if you could add some doxygen for this API and document what you know about each of the Booleans. We can help you fill out the details for any you don't understand, but having this API clearly documented would be a real boon to the project and would make it easy for people to use the new API.

Reply:
Okay will do.

Reply:
Added but not all parameters are described as I do not know what the booleans represent and rather not make erroneous guesses.